charts/druid-tasks-exporter/values.yaml

```diff
@@ -0,0 +1,31 @@
+image:
+  repository: "ghcr.io/wiremind/druid-tasks-exporter"
+  pullPolicy: IfNotPresent
+  tag: "v0.1.0"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  name: ""
+
+service:
+  type: ClusterIP
+  port: 8080
+
+deployment:
+  replicaCount: 1
+  autoscaling:
+    enabled: true
+  command: "/druid-tasks-exporter"
+  args:
+    - "-druid-uri"
+    - "http://YOUR-DRUID-URI:8888/druid/v2/sql"
+  podAnnotations: {}
+  podSecurityContext: {}
+  securityContext: {}
+  resources: {}
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
```

Comment:
add support for servicemonitor:
- https://github.com/wiremind/wiremind-helm-charts/blob/main/charts/karma/templates/servicemonitor.yaml
- https://github.com/wiremind/wiremind-helm-charts/blob/main/charts/karma/values.yaml#L63

but without these fields:
```
  additionalLabels
  annotations
```